test/local/routes/session.js

```diff
@@ -125,3 +125,128 @@ describe('/session/destroy', () => {
   })
 
 })
+
+describe('/session/duplicate', () => {
+  let route
+  let request
+  let log
+  let db
+
+  beforeEach(() => {
+    db = mocks.mockDB({
+    })
+    log = mocks.mockLog()
+    const config = {}
+    const routes = makeRoutes({log, config, db})
+    route = getRoute(routes, '/session/duplicate')
+    request = mocks.mockRequest({
+      credentials: {
+        uid: 'foo',
+        createdAt: 234567,
+        email: 'foo@example.org',
+        emailCode: 'abcdef',
+        emailVerified: true,
+        tokenVerified: true,
+        verifierSetAt: 123456,
+        locale: 'en-AU',
+        uaBrowser: 'Firefox',
+        uaBrowserVersion: '49',
+        uaOS: 'Windows',
+        uaOSVersion: '10',
+        uaDeviceType: 'mobile',
+        uaFormFactor: 'frobble'
```

Comment:
I'm always on the lookout for new metasyntactic variable names and this one is a keeper!